libraries/connectionInfo/connectionUtils.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Returns the type of connection.
+ *
+ * @returns {number} - Type of connection.
+ */
+export function getConnectionType() {
+  const connection = navigator.connection || navigator.webkitConnection;
+  if (!connection) {
+    return 0;
+  }
+  switch (connection.type) {
+    case 'ethernet':
+      return 1;
+    case 'wifi':
```

Comment:
Do we want to include "wimax" with this case?

Reply:
Yes, added. And checked for chrome/edge looks like connection.type is not always there but connection.effectiveType has value. Updated to handle that case.